File: 0001 Code Chef/0022 Fitness/implementation/0001 c/fitness.c
#include <stdio.h>

int main(){
    int test;
    scanf("%d",&test);
    while(test > 0){
        int x;
        scanf("%d",&x);
        int distance_walk_per_day = 2*x;
        int distance_walk_per_week = 5 * distance_walk_per_day;
        printf("%d\n",distance_walk_per_week);
        test = test - 1;
    }
    return 0;
}

File: 0001 Code Chef/0022 Fitness/implementation/0006 php/fitness.php
<?php
$test = (int) fgets(STDIN);
while($test > 0){
    $distance = (int) fgets(STDIN);
    $distance_walk_per_day = 2 * $distance;
    $distance_walk_per_week = 5 * $distance_walk_per_day;
    echo "$distance_walk_per_week\n";

    $test = $test - 1;
}
?>